docs/source/workflows/design_workflows.rst

```diff
@@ -182,6 +187,10 @@ A branch has a name, along with any number of design workflows. A DesignWorkflow
 
 When you're done with a branch, it can be archived, removing it from the results of ``list`` and setting the ``archived`` flag.
 ``list_archived`` lists all archived branches in a project. An archived branch can be restored via its unique ID.
+Note that archiving branches are independent of archiving the design workflows contained within it.
+Archiving a branch, by removing it from listing methods, will hide the entire branch from default displays in the web UI.
```

Comment:
This is a bit of a confusing sentence as written. I think dropping `by removing it from listing methods` will help a lot. And we mention above that archiving a branch will hide it from listing, so that information isn't lost.

Reply:
Change made, with the comments from other teammates taken into account below. LMK if things are a bit clearer this time around.